src/nanoemoji/nanoemoji.py

```diff
@@ -273,6 +273,13 @@ def write_config_preamble(nw, font_config: FontConfig):
         )
         nw.newline()
 
+        if font_config.use_zopflipng:
+            nw.rule(
+                "zopflipng",
+                f"{sys.executable} -m zopfli.png -y"
+                f'{" -v" if FLAGS.verbosity > 0 else ""} $in $out',
```

Comment:
Doesn't seem like interpolation is helping much, and nesting a ternary in it is a bit hard to read.

`(" -v" if FLAGS.verbosity > 0 else "") + " $in $out"`

Or at cost of lines but easier to read

```
args = "$in $out"
if FLAGS.verbosity:
  args = "-v " + args
```

Reply:
Done